packit-ci.fmf

```diff
@@ -99,8 +99,8 @@
 
   discover:
     how: fmf
-    url: https://github.com/RedHat-SP-Security/keylime-tests
-    ref: main
+    url: https://github.com/mbestavros/keylime-tests
```

Comment:
left over?

Reply:
@aplanas This is necessary to run updated e2e tests, per @kkaarreell. Once this PR is ready for merge, we'll merge the updated tests and change this pointer back to the mainline test branch.

Reply:
@aplanas See here: https://github.com/RedHat-SP-Security/keylime-tests/pull/256

Reply:
@mbestavros and @kkaarreell what is the general process for merges like this that require a different testing branch? Do we merge this change first and then merge the changes for `RedHat-SP-Security`? If so, we'll need to make sure we do that it quick succession so CI isn't broken. Or is it best to merge those changes first and then update this PR to no longer point to the branch? If that latter is possible I think it would be preferrable.

Reply:
We can merge test update first and then restore ref in this PR. I am just not at work right now.

Reply:
However, Mark's test PR has failing tests ATM

Reply:
Right, we're not ready to merge this yet (and I would like to cut a new incremental release before this anyway) I was just asking about the process when we get to it.

Reply:
@kkaarreell I've just kicked a retest which should work.